src/renderers/svg/sigma.svg.hovers.def.js

```diff
@@ -51,10 +51,8 @@
       group.setAttributeNS(null, 'class', settings('classPrefix') + '-hover');
       group.setAttributeNS(null, 'data-node-id', node.id);
 
-      if (typeof node.label === 'string') {
-
+      if (typeof node.label === 'string' && node.lable !== '') {
```

Comment:
Hello @qinfchen, you wrote `lable` on this line. Can you change it to `label` please?